server/db/models/containsPaint.js

```diff
@@ -0,0 +1,36 @@
+const dbConnection = require('../db_connection');
+const Recipe = require('./recipe');
+
+class ContainsPaint{
+    constructor({RecipeID, MaterialCode, Grammage, inDB}){
+        this.recipeId = RecipeID;
```

Comment:
I think it's better to reference the instances directly rather than just hold an ID